private-templates-service/client/src/GraphService.tsx

```diff
@@ -1,27 +1,27 @@
-var graph = require('@microsoft/microsoft-graph-client');
+var graph = require("@microsoft/microsoft-graph-client");
 
-function getAuthenticatedClient(accessToken) {
+function getAuthenticatedClient(accessToken: any) {
   // Initialize Graph client
   const client = graph.Client.init({
     // Use the provided access token to authenticate
     // requests
-    authProvider: (done) => {
+    authProvider: (done: any) => {
       done(null, accessToken.accessToken);
     }
   });
 
   return client;
 }
 
-export async function getUserDetails(accessToken) {
+export async function getUserDetails(accessToken: any) {
   console.log(accessToken);
   const client = getAuthenticatedClient(accessToken);
```

Comment:
Should GraphService be a class, so you can store this in state and not create a new one each time its called?

Reply:
Added a todo, will be a part of adding redux. I think its outside the scope of this PR